backend/experiment/rules/matching_pairs.py

```diff
@@ -117,14 +118,42 @@ def get_matching_pairs_trial(self, session):
             title='Tune twins',
             playback=playback,
             feedback_form=None,
-            result_id=prepare_result('matching_pairs', session),
+            result_id=prepare_result('matching_pairs_board', session),
```

Comment:
Why is this renamed?

Reply:
This name was changed to indicate that this result is about the whole board. I'm actually wondering whether to get rid of this result and its handling in `calculate_score` altogether, as it's not strictly necessary to log this when clearing the board.

Reply:
I removed the creation of this result, and the `calculate_score` function altogether.